### DISCUSSION TITLE
Sugestão, se puderem dar uma olhada.

### DISCUSSION THREAD

**Discussion by Aldebaramyr** [YuriAlessandro/gerador-ficha-tormenta20#295] [Geral]
Seria muito legal se após gerar a ficha houve-se como alterar coisas que eu não gostei nela.
Tipo gerei uma ficha de Osteon (Bucaneiro) porem não gostei das armas, gostaria de poder mudar essas opções após gerar a ficha.
Pericias, armas ou habilidades que não sejam próprias da raça ou da classe em questão.

**Comment by YuriAlessandro** [comment]
Cara, para o momento (até mesmo médio-prazo) eu não tenho interesse em fazer isso. A ideia é a ficha ser aleatória, e se você quiser alterar, usar ela como base e criar uma ficha nova.

Permitir esse tipo de alteração vai dar muito trabalho para a gente, e ainda tá faltando terminar coisas básicas (como fichas em níveis maiores, que está perto de ser concluído). 

Quem sabe no futuro.

**Reply by Aldebaramyr** [reply]
Intendo, mas mesmo assim obrigado por disponibilizar esse recurso para gente.
Eu tenho usado muito o app do celular mas ele quando fecha a ficha esconde muitas informações da própria.
